routing/result_interpretation.go

```diff
@@ -335,6 +335,19 @@ func (i *interpretedResult) processPaymentOutcomeIntermediate(
 	case *lnwire.FailUnknownNextPeer:
 		reportOutgoing()
 
+	// Some implementations use this error when the next hop is offline, so we
+	// do the same as FailUnknownNextPeer and process the channel update.
+	case *lnwire.FailChannelDisabled:
+		
+		// Set the node pair for which a channel update may be out of
+		// date. The second chance logic uses the policyFailure field.
+		i.policyFailure = &DirectedNodePair{
```

Comment:
Nice work adding  the`policyFailure` entry here to ensure we'll actually re-try them w/o penalization!